README.md

```diff
@@ -44,13 +45,15 @@ COMMANDS:
          --host value       use customized hostname of srun4000
          --insecure         use http instead of https
          --keep-online, -k  keep online after login
+         --ac-id value      use specified ac_id (default: 0)
```

Comment:
`default: 0` is not wanted as we automatically get `ac_id` AND if it failed the default value is 1, not 0.
It is tested that with `IntFlag` this `default: 0` indication would occur.
We CAN change from `IntFlag` back to `StringFlag` since we do not concern about the real value of `ac_id`.

SHOULD change README with the following things.
1. In the example json config, `acId` as a string should be added.
2. Why `acid` is needed should be explained

Not related comment:
It is scheduled that changes made in [v1.9.7](https://github.com/z4yx/GoAuthing/tree/v1.9.7) should have a json config flag and corresponding documentation.